src/config.ts

```diff
@@ -224,3 +224,11 @@ export const ENABLED_QUICK_ACTIONS = (
 export const FETCH_FIATCONNECT_QUOTES = true
 
 export const WALLETCONNECT_UNIVERSAL_LINK = 'https://valoraapp.com/wc'
+
+export const ENABLED_NETWORK_IDS = [
```

Comment:
I guess we've stopped using alfajores / testnet builds for a while. Should we eventually drop the testnet / mainnet based config we have throughout the app (e.g., networkConfig)

Reply:
oh yes you're right, i didn't think about this carefully because this is temporary (should eventually be passed in via `createApp`) but i will move it to the env files to not break the testnets...even though we don't really use them, it feels like this PR should not be the one that drops them